docs/federation/lookup/README.md

```diff
@@ -6,7 +6,7 @@ This document explains how to configure MinIO with `Bucket lookup from DNS` styl
 
 ### 1. Prerequisites
```

Comment:
IIRC we deprecated federation entirely - should we not just remove this page?

Reply:
This is just a README.md, we have deprecated however the support is not removed for existing deployments and we may never remove it.

Reply:
Leaving the page for a reference for existing deployments.
There is already a note about it being deprecated.